compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -312,6 +313,25 @@ def start_endpoint(*, ep_dir: pathlib.Path, **_kwargs):
     )
 
 
+@app.command(name="login", help="Manually log in with Globus")
+@click.option(
+    "--force",
+    is_flag=True,
+    help="Log in even if already logged in",
+)
+def login(force: bool):
+    if is_client_login():
+        log.info("Don't need to log in when client credentials are present")
+        sys.exit(0)
+
+    lm = LoginManager()
+    if force or lm._token_storage.get_by_resource_server().keys() != lm.SCOPES.keys():
+        # if not forced, only run login flow if any tokens are missing
+        lm.run_login_flow()
+    else:
+        log.info("Already logged in")
```

Comment:
Should we be consistent with the --logout option and hint that --force will force it?  ie.

logout adds "\nPlease use logout --force to proceed"

Reply:
Good point, added